controlplane/internal/controller/openshiftassistedcontrolplane_controller.go

```diff
@@ -182,6 +186,12 @@ func (r *OpenshiftAssistedControlPlaneReconciler) Reconcile(ctx context.Context,
 	k8sVersion, err := r.OpenShiftVersion.GetK8sVersionFromReleaseImage(ctx, releaseImage, oacp)
 	markKubernetesVersionCondition(oacp, err)
 	oacp.Status.Version = k8sVersion
+	if err := r.setOpenshiftVersionInStatus(ctx, oacp); err != nil {
```

Comment:
I think `setOpenshiftVersionInStatus` doesn't clearly say where this version is coming from. What about:
```
oacp.Status.DistributionVersion, err = r.getWorkloadClusterVersion(...)
```

Wouldn't this be clearer?

Reply:
Yeah good idea!